src/sb_pointcloud_processing/launch/line_extractor_rviz_degree_3.launch

```diff
@@ -0,0 +1,23 @@
+<launch>
```

Comment:
We should add a comment to the top of this file explaining what it's doing and why it's here (for demo purposes? for testing?).

Reply:
done